src/_data/whats-new.yml

```diff
@@ -4,8 +4,208 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Mar 30 08:59:14 2020
+updated: Wed Apr 22 10:36:00 2020
 entries:
+- description: Added 'Set a fixed price for bundled products' to the [REST usage notes](https://devdocs.magento.com/guides/v2.3/rest/notes.html)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 20, 2020
+  link: https://github.com/magento/devdocs/pull/7035
+- description: Added correct db schema information in the [Configure declarative schema](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.html)
+    topic
+  versions: 2.3.x
+  type: Technical
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7045
+- description: Added a new reference topic for the [system.xml](https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-systemxml.html)
+    configuration file.
+  versions: 2.3.x
+  type: New Topic
+  date: April 16, 2020
+  link: https://github.com/magento/devdocs/pull/7024
+- description: 'New Page Builder topic on how to configure three different options
+    for styling elements in your content types: [How to configure styling options](https://devdocs.magento.com/page-builder/docs/how-to/how-to-configure-styling-options.html).'
+  versions: 2.3.x
+  type: Technical
+  date: April 15, 2020
+  link: https://github.com/magento/devdocs/pull/7044
+- description: Added a Troubleshooting section in the [Docker services container](https://devdocs.magento.com/cloud/docker/docker-containers-service.html)
+    topic
+  versions: 2.x
+  type: Technical
+  date: April 15, 2020
+  link: https://github.com/magento/devdocs/pull/7043
+- description: Added the 'Metadata interfaces' section to the [Service contract design
+    patterns](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/service-contracts/design-patterns.html#metadata-interfaces)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 14, 2020
+  link: https://github.com/magento/devdocs/pull/6827
+- description: Added the 'Example of an API interface implementation' section to the
+    [Public interfaces & APIs](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/api-concepts.html)
+    topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 14, 2020
+  link: https://github.com/magento/devdocs/pull/6826
+- description: Added the 'Using cUrl in Magento' section to the [Use cURL to run the
+    request](https://devdocs.magento.com/guides/v2.3/get-started/gs-curl.html) topic.
+  versions: 2.3.x
+  type: Major Update
+  date: April 14, 2020
+  link: https://github.com/magento/devdocs/pull/6823
+- description: Created new topic about the [Magento Privacy JS Library](https://devdocs.magento.com/compliance/privacy/magento-privacy-js-library.html)
```

Comment:
Missing endpoint at the end of sentence.

Reply:
```suggestion
- description: Created new topic about the [Magento Privacy JS Library](https://devdocs.magento.com/compliance/privacy/magento-privacy-js-library.html).
```